steps/apptestai-test/0.0.2/step.yml

```diff
@@ -0,0 +1,50 @@
+title: apptestai-test
+summary: |
+  bitrise step for apptest.ai test execution
+description: |
+  bitrise step for apptest.ai test execution.
+
+  Input your apptest.ai Access Key, Project ID, Package file
+  refer to more information from https://app.apptest.ai/#/main/integrations
+
+  We recommend to use Access Key using bitrise workflow secret variable.
+website: https://github.com/forcemax/bitrise-step-apptestai-test
+source_code_url: https://github.com/forcemax/bitrise-step-apptestai-test
+support_url: https://github.com/forcemax/bitrise-step-apptestai-test/issues
+published_at: 2020-03-26T19:42:33.499296+09:00
+source:
+  git: https://github.com/forcemax/bitrise-step-apptestai-test.git
+  commit: 2d86f08403fac96772a670f57614eb3cd780a658
+host_os_tags:
+- osx-10.14
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/forcemax/bitrise-step-apptestai-test
+deps: {}
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- access_key: ci_support:4b55ec5999ea636b0aafb402816ac50b
```

Comment:
Key value must not be here.

Reply:
The value is an example value. Are you saying you want to remove the default value?

Reply:
Yes. Default values are not intended to be examples.
If there are many keys/ids in this service and users may be confused you can add an example in description.